src/message_bus.zig

```diff
@@ -1,12 +1,8 @@
 const std = @import("std");
-const builtin = @import("builtin");
 const assert = std.debug.assert;
 const mem = std.mem;
-const os = std.os;
 const posix = std.posix;
 
-const is_linux = builtin.target.os.tag == .linux;
-
```

Comment:
Killing these two imports is the point! We still need posix for 

```
            const fd = try io.open_socket(address.any.family, posix.SOCK.STREAM, posix.IPPROTO.TCP);
```

I considered sinking that inside IO, so that it has `open_tcp_socket`, but figured that it's fine as-is.